crates/view/src/client.rs

```diff
@@ -709,6 +709,7 @@ where
                 tonic::Request::new(pb::OwnedPositionIdsRequest {
                     trading_pair: trading_pair.map(TryInto::try_into).transpose()?,
                     position_state: position_state.map(TryInto::try_into).transpose()?,
+                    ..Default::default()
```

Comment:
bit of a nit, but I would prefer that we explicitly mark the keys that we're defaulting, so that we don't forgot to edit this file if we add a field that *does* need to be filled.